docs/data-sources/ec_stack.md

```diff
@@ -24,41 +24,41 @@ data "ec_stack" "latest_patch" {
 ```
 
 ## Argument Reference
-s
-* `version_regex` (Required) - Regex to filter the available stacks. Can be any valid regex expression, when multiple stacks are matched through a regex, the latest version is returned. `"latest"` is also accepted to obtain the latest available Stack version.
-* `region` (Required) - Region where the stack pack is, `"ece-region` needs to be used for ece.
-* `lock` (Optional) - Lock the `"latest"` `version_regex` obtained so that when there's a new stack release, the new release doesn't cascade the changes down to the deployments, can be changed at any time.
+
+* `version_regex` (Required) - Regex to filter the available stacks. Can be any valid regex expression, when multiple stacks are matched through a regex, the latest version is returned. `"latest"` is also accepted to obtain the latest available stack version.
+* `region` (Required) - Region where the stack pack is. For Elastic Cloud Enterprise (ECE) installations, use `"ece-region`.
+* `lock` (Optional) - Lock the `"latest"` `version_regex` obtained, so that the new stack release doesn't cascade the changes down to the deployments. It can be changed at any time.
 
 ## Attributes Reference
 
 ~> **NOTE:** Depending on the stack version, some values may not be set. These will not be available for interpolation.
 
 * `version` - The stack version.
-* `accessible` - Whether or not this version is accessible by the calling user. This is only relevant in EC (SaaS) and is not sent in ECE.
-* `min_upgradable_from` - The minimum version recommended to upgrade to this version.
-* `upgradable_to` - The Stack versions that this version can upgrade to.
-* `allowlisted` - Whether or not this version is in the allowlist. This is only relevant in EC (SaaS) and is not sent in ECE.
-* `apm` - Information for APM workloads on this Stack version.
+* `accessible` - To have this version accessible/not accessible by the calling user. This is only relevant for Elastic Cloud ESS, not for ECE.
```

Comment:
In other parts of the documentation we refer to it as "Elasticsearch Service", should we keep it here as well? Or should we change the other ones?

```suggestion
* `accessible` - To have this version accessible/not accessible by the calling user. This is only relevant for Elasticsearch Service (ESS), not for ECE.
```